.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: Release
+on:
+  push:
+    tags:
+    - 'v*'
+jobs:
+  activate:
+    if: github.repository == 'asciidoctor/jekyll-asciidoc'
+    runs-on: ubuntu-latest
+    steps:
+    - run: echo ok go
+  build:
+    needs: activate
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Install Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: '3.0'
+    - name: Configure Bundler
+      run: |
+        bundle config --local path .bundle/gems
+        bundle config --local without coverage docs
+    - name: Install dependencies
+      run: bundle --jobs 3 --retry 3
+    - name: Run tests
+      run: bundle exec rake test:all
+      #comment out the publish step until I understand this better!!
+#    - name: Publish gem
+#      uses: dawidd6/action-publish-gem@v1
```

Comment:
I would recommend to use commands: https://github.com/Mogztter/asciidoctor-kroki/blob/01a8e01da5ed30ade000cf8b72ad927f60246711/.github/workflows/release-ruby.yml#L33-L48
It only takes a handful of commands to publish to rubygems.org reliably (and it will be easier to troubleshoot)